### PR TITLE
feat(hero.js): create the hero banner

### LINKED ISSUE #30: Home: Hero
Designs can be found here

https://www.figma.com/file/6wg9cMORA4odapDwI6Ab1R/Untitled?node-id=528%3A1

The important parts are the bg, title, and sign up button. Don't worry too much about the icons.

Do try and brainstorm ideas we can discuss for animation/interactivity. (scroll listeners, intersection observer, etc)

Sign up will not do anything for the moment. It will most likely be a link instead of a button as well.

### PR DESCRIPTION
**Pull request by CodeItQuick**
The hero banner has to be created for the site.

Closes #30

### CODE REVIEW

File: src/components/layout.js [hunk 1]
```diff
@@ -45,6 +46,7 @@ const _Layout = ({ children }) => {
         />
       </Helmet>
       <Header siteTitle={data.site.siteMetadata.title} />
+      <Hero />
```
Comment (cmgriffing) [inline src/components/layout.js]: Hero doesnt go in the layout. We don't necessarily want it on every page. We want to include it at the page/route level.

File: src/components/hero.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        childImageSharp {
+          fluid(maxWidth: 1440) {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+    }
+  `)
+
+  console.log(data)
```
Comment (cmgriffing) [inline src/components/hero.js]: Probably want to clean up leftover console.logs

File: src/components/hero.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    position: relative;
+    top: 0;
+    margin: 0 auto;
+    background-image: url('${({ image }) => image}');
+    background-size: cover;
+    filter: blur(27px);
+    min-height: 50vh;
+    z-index: -1;
+    width: 100%;
+    max-width: 1340px;
```
Comment (cmgriffing) [inline src/components/hero.js]: Where is this coming from? Why do we want to cap the hero width?

File: src/components/hero.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
+import { useStaticQuery } from "gatsby"
```
Comment (cmgriffing) [inline src/components/hero.js]: Wasnt sure where to put this but we probably want the bg to be page specific. So we would pass the value into this component as a prop.

File: src/components/layout.js (hunk 1 above)
Comment (CodeItQuick) [reply src/components/layout.js]: Moved it to index only. I had a feeling it was in the wrong spot.

File: src/components/hero.js (hunk 2 above)
Comment (CodeItQuick) [reply src/components/hero.js]: cleaned this up

File: src/components/hero.js (hunk 3 above)
Comment (CodeItQuick) [reply src/components/hero.js]: removed the max width

File: src/components/hero.js (hunk 4 above)
Comment (CodeItQuick) [reply src/components/hero.js]: Passed this in as a child, as you said on stream from what I gathered.

**Comment by CodeItQuick** [comment]
Remove the unused library for react-icons
make the h2 tag a class

**Comment by CodeItQuick** [comment]
Rename image and don't use non-static query, do it similar to: https://github.com/gatsbyjs/gatsby/blob/master/examples/using-gatsby-image/src/pages/index.js

**Pull request merged**